Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.csproj

```diff
@@ -3,82 +3,75 @@
   <PropertyGroup>
```

Comment:
@andrewleader mind reviewing these project file changes for notifications?

Reply:
The logic is the same as before, only rearranged some lines to improve readability.